components/dropdown/dropdown-content-mixin.js

```diff
@@ -469,37 +469,42 @@ export const DropdownContentMixin = superclass => class extends RtlMixin(supercl
 		this._width = null;
 		await this.updateComplete;
 
+		const openerPosition = window.getComputedStyle(opener, null).getPropertyValue('position');
 		const boundingContainer = getBoundingAncestor(target);
 		const boundingContainerRect = boundingContainer.getBoundingClientRect();
 
+		// position check in case consuming app (LMS) has overriden position to make content absolute wrt document
+		const bounded = (openerPosition === 'relative' && boundingContainer !== document.documentElement);
```

Comment:
As mentioned in the description, the LMS has special styles to make the opener positioned static, which results in the content absolutely positioned with respect to the document, meaning we can/should opt out of positioning the dropdown content relative to the bounding container (the `d2l-scroll-wrapper`).